src/components/modal/CdrModal.vue

```diff
@@ -92,11 +102,11 @@ let lastActive: Element | null;
 const modalClosed = ref(!props.opened);
 const isOpening = ref(false);
 
-interface offsetValues {
+interface OffsetValues {
```

Comment:
Sonarlint caught this formatting.

Reply:
I'd ask to remove these changes, we use a different config